content/docs/command-reference/repro.md

```diff
@@ -97,6 +93,31 @@ up-to-date and only execute the final stage.
 
 ## Options
 
+- `targets` (optional argument) - one or more file or directory paths (to `.dvc`
```

Comment:
-R is not covered anymore?

Reply:
--glob is not covered anymore

Reply:
Hi! OK I rewrote this as we agreed privately. PTAL